docs/integration.md

```diff
@@ -0,0 +1,248 @@
+# [Integration Guide](#integration-guide)
+The basis of this document is to chowcase the process of integrating `bridget` into an existing web delivery. The end-goal of this module is adding bridging support to the website when browsed through the context of a mobile app via a WebView.
+
+## [Prerequisites](#prerequisites)
+
+This integration guide assumes you already have a web delivery that serves content & a mobile app is being setup for that  particular web delivery.
+
+## [Requirements](#requirements)
+
+#TODO - write requirements that the web delivery needs to satisfy in order for the module to work
+
+## [How to install](#how-to-install)
+> The module is designed to run in the browser. It can also be used in a server (node) environment via npm.
+
+### [Installation via script tag](#install-via-script-tag)
+```html
+<script src="https://unpkg.com/@forward-distribution/bridget">
+  // The module will bootstrap itself when imported, if it is in the correct context (mobile webview)
+</script>
+```
+Put the above script tags in between the <head> tags of your HTML page. It will attach a global `bridget` object. You can access it either via `bridget` or `window.bridget`.
+
+## [Alternative installation via NPM](#alternative-installation-via-npm)
+The module is available on NPM, if you’re using a front-end packager like Browserify or Webpack:
+
+Just run
+
+```
+npm install -g @forward-distribution/bridget
+```
+
+You can then `require` or `import` the lib like any other module.
+
+## [Internals](#internal)
+### Reacting to user actions
+Actions are something that a user performed on your website such as “*Clicked on a teaser*”, "*Navigated to a page/department*", or “*Attempted to share*”.
+
+In the context of your webpage, the browser itself or your custom JavaScript code handles these user events, and reacts to them accordingly. 
+
+In the mobile context, however, these actions need to be propagated and offloaded from the browser (webview) and sent onto the mobile app to be handled natively. 
+
+This is where you, as an integrator, will need to have bridget integrated, so these specific actions can  be handled (see [api reference](#api)).
+
+As an example, internally in the bridge, handling an `onclick` event originating from an `<a>` tag, would resemble something like:
```

Comment:
I don't think we should give this details out for the integrator. We state what happens, not how we do it. We might expect examples like this in some advanced section, but they still will be more in the context of solving some use case and what. we expose as public api (navigateToDoc) and not how we are doing it for the usual use case.